chat-plugins/info.js

```diff
@@ -636,6 +646,33 @@ var commands = exports.commands = {
 		"The parameter 'mega' can be added to search for Mega Evolutions only, and the parameters 'FE' or 'NFE' can be added to search fully or not-fully evolved Pokemon only.",
 		"The order of the parameters does not matter."],
 
+	randpoke: 'randompokemon',
+	randompokemon: function (target) {
+		var targets = target.split(",");
+		var targetsBuffer = [];
+		var qty;
+		for (var i = 0; i < targets.length; i++) {
+			if (!targets[i]) continue;
+			var num = Number(targets[i]);
+			if (Number.isInteger(num)) {
+				if (qty) return this.sendReply("Only specify the number of Pokemon once.");
+				qty = num;
+				if (1 > qty || qty > 15) return this.sendReply("Number of random Pokemon must be between 1 and 15.");
+				targetsBuffer.push("random" + qty);
+			} else {
+				targetsBuffer.push(targets[i]);
+			}
+		}
+		if (!qty) targetsBuffer.push("random1");
+		//Reassigns the target variable to be called in /dexsearch
+		var argumentsBuffer = arguments;
+		argumentsBuffer["0"] = targetsBuffer.join(",");
```

Comment:
I don't really like that idea because then `!dexsearch random1` would be a valid command. If a user saw that broadcast they might think that is the proper syntax for rolling random pokemon which cause problems because: 1) it doesn't make any sense for only one parameter to be dependent on order. 2) dexsearch should be seen as a search tool, not a roller 3) if the random number argument wasn't dependent on position it could be thrown in somewhere in the middle and might go unnoticed, for example `/dexsearch pink, psychic type, random9` might just seem like a dexsearch that returned about the number of results that you would have expected so at a glance the fact that it's a random search might have been missed.

Now I know it might be a bit ridiculous to think that people might broadcast `!dexsearch random6` to try and be confusing, but if it's possible to remove any potentially confusing chat interface with some ugly code, I'd rather have the ugly code then potential bad interface.